integration-cli/docker_api_containers_test.go

```diff
@@ -879,11 +879,14 @@ func (s *DockerSuite) TestContainerApiBadPort(c *check.C) {
 	config := map[string]interface{}{
 		"Image": "busybox",
 		"Cmd":   []string{"/bin/sh", "-c", "echo test"},
-		"PortBindings": map[string]interface{}{
-			"8080/tcp": []map[string]interface{}{
-				{
-					"HostIP":   "",
-					"HostPort": "aa80",
+		"HostConfig": map[string]interface{}{
+			"NetworkMode": "default",
```

Comment:
Ideally, this must be a backward compatible change and hence there shouldnt be a need for changes in Integration-CLI tests. Am I missing something here on why we need to pass the networkmode for a few test-cases ?